semantics.json

```diff
@@ -22,12 +22,20 @@
     "common": true
   },
   {
-    "label": "Feedback text",
+    "label": "Correct feedback text",
     "name": "feedback",
     "type": "text",
     "default": "You got @score of @total points",
-    "description": "Feedback text, variables available: @score and @total. Example: 'You got @score of @total points.'",
-    "common": true
+    "description": "Feedback text when correct answer is given, variables available: @score and @total. Example: 'You got @score of @total points.'",
+    "common": false
+  },
+  {
+    "label": "Incorrect feedback text",
+    "name": "incorrect_feedback",
+    "type": "text",
+    "default": "Sorry, this answer is incorrect",
```

Comment:
I think default text should tell the score. I think it is fine to keep the default text that was here previously, and let authors customize this if they want. i.e. change to  "You got @score of @total points".